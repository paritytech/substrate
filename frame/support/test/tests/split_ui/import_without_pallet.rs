#![cfg_attr(not(feature = "std"), no_std)]

use frame_support::pallet_macros::*;

#[pallet_section]
mod storages {
	#[pallet::storage]
	pub type MyStorageMap<T: Config> = StorageMap<_, _, u32, u64>;
}

#[import_section(storages)]
pub mod pallet {
	
}

fn main() {
}
