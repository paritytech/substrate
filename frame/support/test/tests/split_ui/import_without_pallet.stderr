error: `#[import_section]` can only be applied to a valid pallet module
  --> tests/split_ui/import_without_pallet.rs:12:9
   |
12 | pub mod pallet {
   |         ^^^^^^
